### ISSUE TITLE
[Rancher2] Restricted Admin - permissions are changed 

### ISSUE THREAD

**Issue by jiaqiluo** [rancher/docs#3005]
**Request Summary:**

We need to update the documentation because the scope of permission for a restricted admin will be changed in the Rancher v2.5.6 release.

Here is a quote from https://rancher.com/docs/rancher/v2.x/en/admin-settings/rbac/global-permissions/#list-of-restricted-admin-permissions
```
The restricted-admin permissions are as follows:
1. Has full admin access to all downstream clusters managed by Rancher.
2. Has very limited access to the local Kubernetes cluster. Can access Rancher custom resource definitions, but has no access to any Kubernetes native types.
3. Can add other users and assign them to clusters outside of the local cluster.
4. Can create other restricted admins.
5. Cannot grant any permissions in the local cluster they don't currently have. (This is how Kubernetes normally operates)
```

2 and 5 will not apply anymore because the access to the local cluster is removed. 

I don't have a full list of changes yet. I will keep this issue updated.

**Comment by SheilaghM** [comment]
Feature has been reverted from 2.5.6, so do not merge.  Will be due in 2.5.7.

**Comment by catherineluse** [comment]
Sheilagh said Prachi has a matrix about how we expect restricted admin to work, and that should go in the docs.

**Comment by btat** [comment]
Closed via #3370
